infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```diff
@@ -7,6 +7,9 @@ type policyAssignmentSovereigntyGlobalOptionsType = {
 
   @sys.description('The list of locations that your organization can use to restrict deploying resources to. If left empty, only the deployment location will be allowed.')
   parListOfAllowedLocations: string[]
+
+  @sys.description('The effect type for the Sovereignty Baseline - Global Policies Assignment.')
+  parPolicyEffect: ('Audit' | 'Deny' | 'Disabled' | 'AuditIfNotExists')
```

Comment:
This is to broad as a parameter name and will cause confusion also seems a duplicate of `parTopLevelSovereignGlobalPoliciesEnable`?

Please can we align and make this simplified and clear to customers what the parameter does

Reply:
@oZakari How do you think?